cypress/pages/Registpage.js

```diff
@@ -47,105 +56,116 @@ export const Registpage = {
         cy.get(this.DDL_DOB).click();
         cy.get(this.DDL_DOB_MONTH).select(month);
         cy.get(this.DDL_DOB_YEAR).select(year);
-        cy.get(this.DDL_DOB_DAY).click();
+        cy.get(this.DDL_DOB_DAY+0+date).first(date).click();
         return this;
     },
 
     chooseSubjects(subjects) {
-        cy.get(this.TXT_SUBJECTS).type(subjects).click({force: true});
+        cy.get(this.TXT_SUBJECTS).type(subjects).type('{enter}');
         return this;
     },
 
     selectHobbies(hobby) {
-        cy.get(this.CHK_HOBBIES).click({force: true});
+        if (hobby == "Sport") {
```

Comment:
```js
CHK_HOBBIES: "#hobbiesWrapper label[for]",

selectHobbies(hobbies) {
    hobbies.forEach((hobby) => {
      cy.get(this.CHK_HOBBIES).contains(hobby).click();
    });
    return this;
},
```
No matter what your test data has one or more hobbies, place them in an list for general usage